### ISSUE TITLE
lol_dba/sql_generator (LoadError)

### ISSUE THREAD

**Issue by enthusiasmus** [plentz/lol_dba#103]
After adding the Gem to the Gemfile and executing bundle install, I tried `lol_dba db:find_indexes`, but received following error:

```
root@12b7b0cb50f4:/usr/src/app# lol_dba db:find_indexes
Traceback (most recent call last):
	7: from /usr/local/bundle/bin/lol_dba:23:in `<main>'
	6: from /usr/local/bundle/bin/lol_dba:23:in `load'
	5: from /usr/local/bundle/gems/lol_dba-2.1.7/bin/lol_dba:8:in `<top (required)>'
	4: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	3: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
	2: from /usr/local/bundle/gems/lol_dba-2.1.7/lib/lol_dba/cli.rb:2:in `<top (required)>'
	1: from /usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require'
/usr/local/lib/ruby/2.6.0/rubygems/core_ext/kernel_require.rb:54:in `require': cannot load such file -- lol_dba/sql_generator (LoadError)
```

any ideas? 
Thanks in advance! 
Best regards,
Lukas

**Comment by plentz** [comment]
Fixed by #99

**Comment by enthusiasmus** [comment]
That was fast! Thanks!
